### DISCUSSION TITLE
After installation and startup, nothing happens

### DISCUSSION THREAD

**Discussion by w0wd1ck** [hacksider/Deep-Live-Cam#236] [Q&A]
Hi friends. installed everything as written in the guide and also watched this video https://www.youtube.com/watch?v=MICScLVM1x8.
but unfortunately after startup nothing happens, help please

**Comment by w0wd1ck** [comment]
![IMG_20240813_032814_599](https://github.com/user-attachments/assets/1afcb067-7e5b-4f2b-af17-3a5652f59b5a)

**Reply by vlednerk** [reply]
Something wrong with python

**Comment by MR-VL** [comment]
try running with by using "py run.py" instead of python run.py. Sometimes windows looks for this cmd instead of python
